### PR TITLE
Update tooling packages to version 13.

### PR DESCRIPTION
**Pull request by alan-agius4**
This is a followup on #43343

### CODE REVIEW

**Comment by alan-agius4** [comment]
Looks like the dev-server is not being killed correctly as the port is not being released.

**Comment by alan-agius4** [comment]
```

Error: listen EADDRINUSE: address already in use 127.0.0.1:8080
    at Server.setupListenHandle [as _listen2] (net.js:1320:16)
    at listenInCluster (net.js:1368:12)
    at GetAddrInfoReqWrap.doListen [as callback] (net.js:1505:7)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:71:8) {
  code: 'EADDRINUSE',
  errno: -48,
  syscall: 'listen',
  address: '127.0.0.1',
  port: 8080
}
```

Looks like the weboack-dev-server port 0 is not working correctly.

**Comment by alan-agius4** [comment]
@AndrewKushnir & @petebacondarwin can I get a review for size-checks?

**Review by josephperrott — APPROVED** [review]
LGTM

File: goldens/size-tracking/integration-payloads.json [hunk 1]
```diff
@@ -2,55 +2,57 @@
...
     }
   },
   "cli-hello-world-ivy-minimal": {
     "master": {
       "uncompressed": {
-        "runtime-es2015": 1205,
-        "main-es2015": 17597,
-        "polyfills-es2015": 37127
+        "runtime": 1107,
+        "main": "TODO(i): temporarily increase the payload size limit from 17597 - this needs to be investigate further what caused the increase.",
```
Comment (alan-agius4) [inline goldens/size-tracking/integration-payloads.json:15]: This is not a regression from version 13.0.0. This has actually regressed in version 12.1.0.

**Review by AndrewKushnir — APPROVED** [review]
Reviewed-for: size-tracking

File: goldens/size-tracking/integration-payloads.json (hunk 1 above)
Comment (petebacondarwin) [reply goldens/size-tracking/integration-payloads.json:15]: Can you create an issue to track this investigation?

File: integration/ivy-i18n/package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
     "translated:legacy-xlf:update-translations": "node scripts/update-xlf-translation-file.js \"../tmp/legacy-locales/messages.legacy.xlf\"",
     "translated:legacy-xmb:test": "yarn ng build && yarn extract --format xmb --outputPath ../tmp/legacy-locales/messages.legacy.xmb && yarn translated:legacy-xmb:update-translations && yarn ng build --configuration=translated-legacy-xmb && yarn translated:legacy:translate -t \"../tmp/legacy-locales/messages.legacy.xtb\" && yarn translated:legacy:e2e",
     "translated:legacy-xmb:update-translations": "node scripts/update-xmb-translation-file.js \"../tmp/legacy-locales/messages.legacy.xmb\"",
-    "extract": "localize-extract --source dist/main-es2015*.js"
+    "extract": "localize-extract --source dist/main*.js"
```
Comment (petebacondarwin) [inline integration/ivy-i18n/package.json:30]: 🎉

File: package.json [hunk 3]
```diff
@@ -41,11 +41,11 @@
   },
   "// 1": "dependencies are used locally and by bazel",
   "dependencies": {
-    "@angular-devkit/build-angular": "12.0.4",
-    "@angular-devkit/build-optimizer": "0.1200.4",
-    "@angular-devkit/core": "12.0.4",
-    "@angular-devkit/schematics": "12.0.4",
-    "@angular/cli": "12.0.4",
+    "@angular-devkit/build-angular": "13.0.0-next.6",
```
Comment (petebacondarwin) [inline package.json:44]: Is the plan to bump these versions regularly until we hit 13.0.0?

File: goldens/size-tracking/integration-payloads.json (hunk 1 above)
Comment (alan-agius4) [reply goldens/size-tracking/integration-payloads.json:15]: Yeah sure, I’ll create it later on today

**Review by petebacondarwin — APPROVED** [review]
LGTM as long as we have an issue to track the size regression

Reviewed-for: size-tracking

File: package.json (hunk 3 above)
Comment (alan-agius4) [reply package.json:44]: Yes, it will be done automatically through Renovate.

File: goldens/size-tracking/integration-payloads.json (hunk 1 above)
Comment (alan-agius4) [reply goldens/size-tracking/integration-payloads.json:15]: https://github.com/angular/angular/issues/43568

**Pull request closed without merging**